examples/DatePicker.menu.tsx

```diff
@@ -12,7 +12,11 @@ export default () => {
 
   return (
     <>
-      <IconButton onClick={() => setOpened(!opened)} id='picker-button'>
+      <IconButton
+        onClick={() => setOpened(!opened)}
+        id='picker-button'
+        aria-label='date picker'
```

Comment:
Using `label` prop would work better than `aria-label` as it also shows a tooltip for sighted users. And the contents should be in _Sentence case_.

also (unrelated): the `id` can be removed

Reply:
Changes made [here](https://github.com/iTwin/iTwinUI/pull/1486/commits/4e60dd9c07558fec535d9b0f7180057ca6deff4f). `id` removed, `aria-label` switched to `label`, converted to Sentence case.

Reply:
can you also make the label text more useful? it should consist of the action rather than the name of the component. e.g. "Choose date" or "Pick date" instead of "Date picker"

Reply:
Renamed to "Choose date"